tests/tools/telephony/test_check_extension_status_tool.py

```
@@ -383,3 +419,129 @@ async def send_command_side_effect(method, resource, data=None, params=None):
+        assert out["availability_status"] == "available"
+
+
+class TestGuardrailCustomStates:
+    @pytest.mark.asyncio
+    async def test_configured_custom_state_is_allowed(self, tool_context_factory):
+        ctx = tool_context_factory(
+            extensions={"102": {"dial_string": "PJSIP/102", "transfer": True,
+                                "device_states": [{"id": "Custom:DND102", "status": "dnd"}]}},
+            tool_cfg={"restrict_to_configured_extensions": True})
+        async def sc(method, resource, data=None, params=None):
+            if "Custom" in resource: return {"name": "Custom:DND102", "state": "NOT_INUSE"}
```

Ruff (0.16.1)

[error] Multiple statements on one line (colon)

(E701)